src/com/dynatrace/loadrunner/logic/LoadRunnerConverter.java

```diff
@@ -30,8 +30,10 @@
 	private List<ScriptFile> hFiles;
 	private List<ScriptFile> cFiles;
 	private boolean mode;
+	private String scriptName;
+	private boolean cEngine;
 
-	private void replaceOriginalFile(File sourceFile, File targetFile) throws IOException{
+	private void replaceOriginalFile(File sourceFile, File targetFile) throws IOException {
```

Comment:
format...
mark all the *.java files in idea, and pick 'Reformat Code' from context menu